index.js

```diff
@@ -28,30 +30,36 @@ Toolkit.run(async tools => {
     return
   }
 
-  const majorWords = process.env['INPUT_MAJOR-WORDING'].split(',')
-  const minorWords = process.env['INPUT_MINOR-WORDING'].split(',')
-  const preReleaseWords = process.env['INPUT_RC-WORDING'].split(',')
+  let majorWords, minorWords, preReleaseWords, patchWords;
 
-  // if patch words aren't specified, any commit message qualifies as a patch
-  const patchWords = process.env['INPUT_PATCH-WORDING'] ? process.env['INPUT_PATCH-WORDING'].split(',') : null
+  // handle empty or null value
+  try {
+    majorWords = (process.env['INPUT_MAJOR-WORDING'] || null).split(',');
+    minorWords = (process.env['INPUT_MINOR-WORDING'] || null).split(',');
+    preReleaseWords = (process.env['INPUT_RC-WORDING'] || null).split(',');
+    // if patch words aren't specified, any commit message qualifies as a patch
+    patchWords = (process.env['INPUT_PATCH-WORDING'] || null).split(',');
+  } catch (e) {
+    // do nothing
```

Comment:
@ryan-cha when for example process.env['INPUT_MAJOR-WORDING'] is empty it would throw immediately, not checking minor, preRelease, patch words.
Is that intended?

Reply:
@phips28 Oh, my bad. 
Of course, it's not what I intended. I'll take some time to correct this PR soon!